apps/demo/index.html

```diff
@@ -7,11 +7,13 @@
     <title>H5Web</title>
 
     <meta name="description" content="Web-based HDF5 file viewer." />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
-    <link rel="apple-touch-icon" href="%PUBLIC_URL%/favicon192.png" />
+    <link rel="icon" href="/favicon.ico" />
+    <link rel="apple-touch-icon" href="/favicon192.png" />
+    <script src="https://polyfill.io/v3/polyfill.min.js?features=globalThis"></script>
   </head>
   <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
     <div id="root"></div>
+    <script type="module" src="/src/index.tsx"></script>
```

Comment:
Contrary to CRA, Vite's entrypoint is not `index.tsx`, but `index.html` (just like a browser), so we need to reference `index.tsx` here so Vite can find it a process it.